core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/dot-ema-shell.component.ts

```diff
@@ -174,6 +178,9 @@ export class DotEmaShellComponent implements OnInit, OnDestroy {
     private readonly activatedRoute = inject(ActivatedRoute);
     private readonly router = inject(Router);
     private readonly siteService = inject(SiteService);
+    private readonly dotMessageService = inject(DotMessageService);
```

Comment:
@zJaaal According to recents conversations in Slack, all private attr should start with `#`, like this `#dotMessageService.`